tox.ini

```diff
@@ -7,127 +7,58 @@
 ; the package installation).
 [tox]
 envlist = bandit, black, black-check, isort, isort-check, copyright_check, docs, flake8, liccheck, mypy, py{3.6,3.7,3.8,3.9,3.10}, dependencies_check, plugins_deps
-
+isolated_build = True
 
 [testenv]
 basepython = python3
-whitelist_externals = /bin/sh
+whitelist_externals = 
+    /bin/sh
+    poetry
 passenv = *
 extras = all
-deps =
-    aiohttp>=3.7.4,<3.8
-    aioprometheus>=20.0.0,<21.0.0
-    black==19.10b0
-    colorlog==4.1.0
-    defusedxml==0.7.1
-    docker==4.2.0
-    gym==0.15.6
-    isort==5.7.0
-    mistune==2.0.0a4
-    numpy>=1.18.1
-    oef==0.8.1
-    openapi-core==0.13.2
-    openapi-spec-validator==0.2.8
-    pexpect==4.8.0
-    pytest==7.0.0
-    pytest-asyncio==0.16.0
-    pytest-cov==3.0.0
-    pytest-custom-exit-code==0.3.0
-    pytest-randomly==3.10.3
-    pytest-rerunfailures==10.2
-    scikit-image>=0.17.2
-    sqlalchemy==1.4.17
-    temper-py==0.0.3
-    yoti==2.14.0
-    ; Plugin dependencies. We need this
-    ; because we use --no-deps to install the plugins.
-    ; aea_ledger_cosmos/aea_ledger_fetchai
-    ecdsa==0.17.0
-    asn1crypto==1.4.0
-    bech32==1.2.0
-    ; aea_ledger_ethereum
-    web3==5.12.0
-    ipfshttpclient==0.6.1
-    eth-account==0.5.2
-    ; for password encryption in cosmos
-    pycryptodome==3.15.0
-    blspy==1.0.14
-	bip-utils==2.5.1
-	python-mbedtls==2.2.0
-	py-sr25519-bindings==0.1.4
-    cosmpy>=0.5.0,<0.6.0
-
-
+deps = toml
 commands =
-	; for some reason tox installs aea without respect to the dependencies version specified in setup.py. at least in CI env
-	; so install current aea in a normal way
-	pip install {toxinidir}[all]
-    python -m pip install --no-deps file://{toxinidir}/plugins/aea-ledger-ethereum
-    python -m pip install --no-deps file://{toxinidir}/plugins/aea-ledger-cosmos
-    python -m pip install --no-deps file://{toxinidir}/plugins/aea-ledger-fetchai
-    python -m pip install --no-deps file://{toxinidir}/plugins/aea-cli-ipfs
-    pytest -rfE tests/ --cov=aea --cov=packages/fetchai/connections --cov=packages/fetchai/contracts --cov=packages/fetchai/protocols --cov=packages/fetchai/skills --cov-report=html --cov-report=xml --cov-report=term --cov-report=term-missing --cov-config=.coveragerc {posargs}
-
-[testenv:py3.6]
-basepython = python3.6
-deps =
-    {[testenv]deps}
-    tensorflow==2.4.0
+    poetry install --only testing --only main --only packages -E all
+    poetry run pip install openapi-core==0.13.2
+   	poetry run pip install {toxinidir}[all]
+    poetry run python ./install_packages.py jsonschema
```

Comment:
Why manually add `openapi-core` and `jsonschema`?

Reply:
required by http server connection. does not work with the latest modules. so need to update package code